react/src/pages/NotificationPopover.tsx

```diff
@@ -66,7 +68,9 @@ export default function NotificationPopover({ lastLoginTime }: NotificationPopov
         <div>
             <IconButton onClick={handlePopoverOpen}>
                 <Tooltip title="See notifications" arrow>
-                    <NotificationsActive fontSize='large' style={{fill: "white"}} />
+                    <Badge badgeContent={numNotification} color="secondary" showZero>
```

Comment:
Looks good! My only suggestion is that the badge should not show if there are no notifications.
```suggestion
                    <Badge badgeContent={numNotification} color="secondary">
```